pennylane/compiler/qjit_api.py

```diff
@@ -377,7 +377,37 @@ def loop_rx(x):
         ops_loader = compilers[active_jit]["ops"].load()
         return ops_loader.while_loop(cond_fn)
 
-    raise CompileError("There is no active compiler package.")  # pragma: no cover
+    # if there is no active compiler, simply interpret the while loop
+    # via the Python interpretor.
+    def _decorator(body_fn):
```

Comment:
```suggestion
    def _decorator(body_fn: Callable) -> Callable:
         """Transform that will call the input ``body_fn`` until the closure variable ``cond_fn`` is met.
         
         Args:
             body_fn (Callable):
         
         Closure Variables:
             cond_fn (Callable):
             
         Returns:
              Callable: a callable with the same signature as ``body_fn`` and ``cond_fn``.
         """
```

Helps a lot for debugging purposes.

Reply:
thanks!